src/UI/Root/Components/Sidebar/Navigation/Navigation.test.tsx

```diff
@@ -11,6 +16,7 @@ import {
   QueryResolverImpl,
   QueryManagerResolverImpl,
 } from "@/Data";
+import * as queryModule from "@/Data/Managers/V2/Compilation/GetCompilerStatus/useGetCompilerStatus";
```

Comment:
It's the first time I see us using this naming convention, is there a specific reason?

Reply:
nope, Do you want me to change it, I chose it spontaneously as it is imported as a whoel module, and it's query so it came out naturally

Reply:
I think it's better to not rename the import :D

Reply:
Oh, that's what you meant. we need to import like this to be able to spy on that module, this convention is already existing in the code base. I think it's the best approach from what I researched